ui/tests/acceptance/build-detail-test.ts

```diff
@@ -1,15 +1,16 @@
+import { currentURL, visit } from '@ember/test-helpers';
 import { module, test } from 'qunit';
-import { visit, currentURL } from '@ember/test-helpers';
+
+import login from 'waypoint/tests/helpers/login';
 import { setupApplicationTest } from 'ember-qunit';
 import { setupMirage } from 'ember-cli-mirage/test-support';
-import login from 'waypoint/tests/helpers/login';
 
 module('Acceptance | build detail', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
-  login();
 
   test('redirects from the ID to the sequence route', async function (assert) {
+    await login();
```

Comment:
Just spitballing, but I wonder if we could add a `setupSession(hooks)` function to the `login` module and use that to reduce the repetition a bit.

```ts
// tests/helpers/login.ts

import { authenticateSession } from 'ember-simple-auth/test-support';

export default async function login(waypointToken = 'default-test-token-value'): Promise<void> {
  return await authenticateSession({ token: waypointToken });
}

export function setupSession(hooks: NestedHooks): void {
  hooks.beforeEach(() => login());
}
```

```ts
// ui/tests/acceptance/build-detail-test.ts

import login, { setupSession } from 'waypoint/tests/helpers/login';
import { setupApplicationTest } from 'ember-qunit';
import { setupMirage } from 'ember-cli-mirage/test-support';
import login from 'waypoint/tests/helpers/login';

module('Acceptance | build detail', function (hooks) {
  setupApplicationTest(hooks);
  setupMirage(hooks);
  setupSession(hooks);
```

Reply:
yeah that's a good idea. Updating

Reply:
done in https://github.com/hashicorp/waypoint/pull/2619/commits/ac7800ebf0fb64c8f4995725004591d9b24a8581